JMS/src/test/java/com/ibm/mq/samples/jms/JmsSubTest.java

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
I don't know why we need Both JmsPubTest and JmsSubTest. They are doing the same thing.

Reply:
Both tests are still in the pull request!